packages/jest/src/schematics/jest-project/files/jest.config.js__tmpl__

```diff
@@ -3,7 +3,8 @@ module.exports = {
   preset: '<%= offsetFromRoot %>jest.config.js',<% if(testEnvironment) { %>
   testEnvironment: '<%= testEnvironment %>',<% } %><% if (supportTsx) { %>
   transform: {
-    '^.+\\.[tj]sx?$': 'ts-jest'
+    '^.+\\.[tj]sx?$': <% if (transformer == 'babel-jest') { %>[ 'babel-jest',
```

Comment:
Why do we need to specify `cwd` here? I'd also try to not create the `babel.config.json` file since it is only used for jest tests -- for normal serve/build the babel config is encapsulated by our builders.

If anything I would inline it in `jest.config.js` so people won't think it controls babel config outside of jest.

Reply:
I had to specify `cwd` because this is the only thing that uses the working dir to find the config file (i.e. otherwise, the path would have to have been `./apps/${appName}/babel.config.json`)

Good point about the config. I'm not sure how possible it will be to provide it differently. I'll have a look at the way `babel-jest` loads it.

At the very least, I could probably give it a different name.

Reply:
I've renamed the file. There's no way to pass the config to `babel-jest` other than telling it what file to load from.